components/System/DarkModeToggle.vue

```diff
@@ -35,18 +35,18 @@ export default Vue.extend({
     }
   },
   computed: {
-    ...mapGetters('displayPreferences', ['getBooleanCustomPref']),
+    ...mapGetters('displayPreferences', ['getDarkMode']),
```

Comment:
This can be a ``mapState`` now that we use the correct datatypes. I'm going to do it myself, just noting this for later so I don't forget about it. ^^